project-base/tests/ShopBundle/Functional/Model/Pricing/InputPriceRecalculationSchedulerTest.php

```diff
@@ -183,7 +183,7 @@ private function doTestOnKernelResponseRecalculateInputPrices(Money $inputPrice,
         $em->refresh($payment);
         $em->refresh($transport);
 
-        $this->assertThat($payment->getPaymentPriceByDomainId(Domain::FIRST_DOMAIN_ID)->getPrice(), new IsMoneyEqual($expectedPrice));
+        $this->assertThat($payment->getPrice(Domain::FIRST_DOMAIN_ID)->getPrice(), new IsMoneyEqual($expectedPrice));
         $this->assertThat($transport->getTransportPriceByDomainId(Domain::FIRST_DOMAIN_ID)->getPrice(), new IsMoneyEqual($expectedPrice));
```

Comment:
I expected you will rename it even in transport :) (get~~Transport~~Price~~ByDomainId~~)

Reply:
Sorry, my mistake, fixed